plumbum/commands/base.py

```diff
@@ -335,8 +335,6 @@ def popen(self, args=(), **kwargs):
 
 
 class BoundEnvCommand(BaseCommand):
-    __slots__ = ("cmd",)
-
```

Comment:
This does need `__slots__ = ("cmd",)`, that's correct. It just doesn't need to redefine slots that are already present in the base class.